doc/api/module.md

```diff
@@ -585,6 +585,9 @@ added: v22.8.0
 <!-- YAML
 added: v8.8.0
 changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/60302
+    description: Change stability index for async variant from Release candidate to Experimental.
```

Comment:
We don't have entries for going from experimental to experimental, I'm not sure it's useful to have it
```suggestion
```